website/src/pages/dashboard.tsx

```diff
@@ -44,9 +46,11 @@ const Dashboard = () => {
         <meta name="description" content="Chat with Open Assistant and provide feedback." key="description" />
       </Head>
       <Flex direction="column" gap="10">
-        <WelcomeCard />
-        <TaskOption content={availableTaskTypes} />
-        <LeaderboardWidget />
+        <ToSWrapper>
+          <WelcomeCard />
+          <TaskOption content={availableTaskTypes} />
+          <LeaderboardWidget />
+        </ToSWrapper>
```

Comment:
I added this wrapper here, although implementation-wise it could be added anywhere within the application.